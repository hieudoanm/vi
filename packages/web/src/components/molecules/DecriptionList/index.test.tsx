import DescriptionList from '.';
import { render } from '@testing-library/react';

describe('DescriptionList', () => {
  test('render default', () => {
    const wrapper = render(<DescriptionList items={[]} />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
