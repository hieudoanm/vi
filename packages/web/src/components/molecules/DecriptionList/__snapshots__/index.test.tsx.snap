// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DescriptionList render default 1`] = `
<div>
  <div
    class="overflow-hidden rounded-lg border shadow"
  />
</div>
`;
